doc/README.id.md

```diff
@@ -0,0 +1,151 @@
+
+Sebagian karena keadaan, dan sebagian karena popularitasnya, YouTube yang paling didukung dari layanan ini. Jika kamu menggunakan atau familiar dengan salah satu layanan ini, tolong bantu kami meningkatkan dukungan untuk mereka! Kami sedang mencari pengelola untuk SoundCloud dan PeerTube.
+
+Jika kamu berniat untuk menambahkan layanan baru, silakan hubungi kami terlebih dahulu! [Dokumentasi](https://teamnewpipe.github.io/documentation) kami memberikan informasi lebih lanjut tentang bagaimana layanan baru dapat ditambahkan ke aplikasi dan ke [Extractor NewPipe](https://github.com/TeamNewPipe/NewPipeExtractor).
+
+## Deskripsi
+
+NewPipe bekerja dengan mengambil data yang diperlukan dari API resmi (misalnya PeerTube) dari layanan yang kamu gunakan. Jika API resmi dibatasi (misalnya YouTube) untuk tujuan kami, atau merupakan hak milik, aplikasi akan mem-parsing situs web atau menggunakan API internal sebagai gantinya. Ini berarti kamu tidak memerlukan akun di layanan apa pun untuk menggunakan NewPipe.
+
+Dan juga, karena mereka adalah perangkat lunak yang gratis dan sumber terbuka, baik aplikasi atau pun Extractor tidak menggunakan pustaka atau kerangka kerja berpemilik, seperti layanan Google Play. Ini berarti kamu dapat menggunakan NewPipe pada perangkat atau ROM khusus yang tidak menginstal aplikasi Google. 
+
+### Fitur
+
+* Tonton video dengan resolusi hingga 4K
+* Dengarkan audio di latar belakang, hanya memuat aliran audio untuk menghemat data
+* Mode popup (pemutar mengambang, alias Picture-in-Picture)
+* Tonton streaming langsung
+* Tampilkan/sembunyikan subtitel/teks tertutup
+* Cari video dan audio (di YouTube, kamu juga dapat menentukan bahasa konten)
+* Buat antrean video (dan secara opsional simpan sebagai playlist lokal)
+* Menampilkan/menyembunyikan informasi umum tentang video (seperti deskripsi dan tag)
+* Tampilkan/sembunyikan video berikutnya/terkait
+* Tampilkan/sembunyikan komentar
+* Cari video, audio, channel, playlist, dan album
+* Jelajahi video dan audio dalam channel
+* Subscribe ke channel (Ya, tanpa login ke akun apa pun!)
+* Dapatkan notifikasi tentang video baru dari channel yang kamu subscribe
+* Buat dan edit grup channel (untuk penelusuran dan pengelolaan yang lebih mudah)
+* Jelajahi umpan video yang dihasilkan dari grup channel kamu
+* Lihat dan telusuri riwayat tontonan kamu
+* Cari dan tonton playlist (ini adalah playlist remote, yang berarti diambil dari layanan yang kamu jelajahi)
+* Buat dan edit playlist lokal (ini dibuat dan disimpan di dalam aplikasi, dan tidak ada hubungannya dengan layanan apa pun)
+* Unduh video/audio/subtitel (teks tertutup)
+* Buka di Kodi
+* Tonton/blokir materi yang dibatasi usia
+
+<!-- Span tersembunyi untuk menjaga agar tautan lama tetap kompatibel. -->
+<span id="updates"></span>

```

Comment:
This is the first time I've ever seen someone translate this comment. Thank you!

However, this comment and the span below it are not needed in translated READMEs, so you should remove them. See https://github.com/TeamNewPipe/NewPipe/pull/8754.